src/components/projects/projects-table/projects-table.module.css

```diff
@@ -3,6 +3,30 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-.table {
-  padding: 0em 2em 0em 2em;
+.container {
+  width: 100%;
+  padding-right: 15px;
```

Comment:
you might be able to shorten this with the single line padding that goes something like `padding: 5px 5px 5px 5px`